alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,34 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces)
+Normal:,,
+all elements,/authors,
+"all ""author"" elements",//author,
+"the root element ""authors"" - 2 variants","/* ,  authors, child::authors",
+"the same as 3,  but this time without using the name of the element",/*,
+all elements enclosed by elements  /authors/author,/authors/author,
```

Comment:
/authors/author returns all "author" elements, but need all elements that are inside of "author"

Reply:
//author/*   ?